shared/actions/fs/index.js

```diff
@@ -619,6 +619,13 @@ function* openPathItem(action: FsGen.OpenPathItemPayload): Saga.SagaGenerator<an
   )
 }
 
+const uploadFanOut = (state: TypedState, action: FsGen.UploadsPayload) =>
+  Saga.sequentially(
```

Comment:
why sequentially vs all?

Reply:
I didn't pick sequentially over all consciously. I guess I just didn't use `all` before and assumed `sequentially` was preferred. Do we prefer `all` when order doesn't matter?

Reply:
yes, do things in parallel